docs/content/en/docs/reference/clusterspec/vsphere.md

```diff
@@ -142,7 +142,7 @@ The Kubernetes version you want to use for your cluster. Supported values: `1.20
 ## VSphereDatacenterConfig Fields
 
 ### datacenter (required)
-The vSphere datacenter to deploy the EKS Anywhere cluster on. For example `SDDC-Datacenter`.
+The vSphere datacenter to deploy the EKS Anywhere cluster on. For example `SDDC-Datacenter`. Note: It's required to add double quotes around the datacenter name in case it has a space in the name of it, example "datacenter one".
```

Comment:
```suggestion
The vSphere datacenter to deploy the EKS Anywhere cluster on. For example `SDDC-Datacenter`. Add double quotes around the datacenter name if there is a space in its name, such as "datacenter one".
```
Small wording suggestions otherwise it looks good to me.